packages/react-native-bpk-component-flat-list/stories.js

```diff
@@ -60,25 +62,28 @@ class StatefulBpkFlatList extends React.Component<
   {
     extraEntries: number,
     showImages: boolean,
+    includeSearch: boolean,
   },
-  { selectedCountry: string },
+  { countries: Array<{ id: string, name: string }>, selectedCountry: string },
 > {
   itemPressCallbacks: { [string]: () => mixed };
 
   static propTypes = {
     extraEntries: PropTypes.number,
     showImages: PropTypes.bool,
+    includeSearch: PropTypes.bool,
   };
 
   static defaultProps = {
     extraEntries: 0,
     showImages: false,
+    includeSearch: false,
   };
 
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
     this.itemPressCallbacks = {};
-    this.state = { selectedCountry: 'DJ' };
+    this.state = { selectedCountry: 'DJ', countries: this.getData() };
```

Comment:
getData depends on a prop so we shouldn't only fetch it in the constructor. We should memoize the function or use https://reactjs.org/docs/react-component.html#static-getderivedstatefromprops

Reply:
Sorry, didn't realise this a `story` file 🤦‍♂ . It's fine here.